bin/nyc.js

```diff
@@ -127,6 +131,16 @@ var yargs = require('yargs/yargs')(process.argv.slice(2))
     type: 'boolean',
     description: 'should nyc handle instrumentation?'
   })
+  .option('hook-run-in-context', {
+    default: false,
+    type: 'boolean',
+    description: 'should nyc wrap vm.runInThisContext?'
+  })
+  .option('hook-create-script', {
+    default: false,
```

Comment:
we've tried to be fairly batteries included so far, for instance we already instrument subprocesses right out of the gate.  I might be tempted to default this to `true`. I'm happy with the argument name 👍